samples/led-matrix-weather/led-matrix-weather.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <RootNamespace>led_matrix_weather</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
+    </ProjectReference>
+    <ProjectReference Include="../../src/Iot.Device.Bindings/Iot.Device.Bindings.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Protobuf Include="weather.proto" />

```

Comment:
mm Didn't you require a package reference to GRPC tools in order for this to work?

Reply:
This actually should be removed since I don't use this anymore